### PR TITLE
Review one of your past projects

### PR DESCRIPTION
**Pull request by DianaBeki**
In the following task, I did the following
- Added index.html file
- Added style.css

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+ul li {
+    margin-right: 50px;
+}
+h1 {
+text-align: center;
+font-size: 3rem;
+margin-top: 10rem;
+
+}
```
Comment (DianaBeki) [inline css/style.css:35]: - Check your linter errors to avoid syntax errors and make your code readable and smartly organized.

**Review by DianaBeki — COMMENTED** [review]
Hi @DianaBeki  
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Good Pr Description✔️ 
- Meaningful commits ✔️


_**Notes To Consider For Your Next Projects**_

- Good Job so far, your commit message is clear and meaningful but kindly make sure in your next projects, The commit message should be committed separately for each file.



 ## Required Changes ♻️


- [x] 🛑 You should use a linter to check on errors before pushing your project.
- [x] 🛑 update readme file with correct LinkedIn issues URL, Add MIT template.




## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @Rana-Mudassir ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- Inline comments are present. ✔

## (Changes Required) Aspects to improve: ♻️
- [ ] **For more changes  Please check the inline comments below.**
## [Optional] suggestions:
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by PinkMoon25 — APPROVED** [review]
Hi @DianaBeki , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Good Pr Description :heavy_check_mark:
Meaningful commits :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] As suggested by the previous reviewer, https://github.com/DianaBeki/landingpage/pull/1#discussion_r1009806620 please always make sure to provide details to the student so it is clear hoe can he improve or what changes he have to make specifically, a good rule of thumb :+1: is "Always try to be helpful"

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**